### PR TITLE
Remove not needed interfaces

### LINKED ISSUE #291: Remove unnecessary "interface" classes

### PR DESCRIPTION
**Pull request by mirpo**
fixes #291

### CODE REVIEW

File: test/integration/tracker-node-status.test.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ describe('check status message flow between tracker and two nodes', () => {
     })
 
     it('tracker should receive status message from node', async (done) => {
-        tracker.protocols.trackerServer.once(TrackerServer.events.NODE_STATUS_RECEIVED, ({ statusMessage, nodeType }) => {
+        tracker.protocols.trackerServer.once(TrackerServer.events.NODE_STATUS_RECEIVED, ({ statusMessage, _ }) => {
```
Comment (harbu) [inline test/integration/tracker-node-status.test.js]: You can leave out underscore.

File: test/integration/tracker-node-status.test.js [hunk 2]
```diff
@@ -30,7 +30,7 @@ describe('check status message flow between tracker and two nodes', () => {
     })
 
     it('tracker should receive status from second node', async (done) => {
-        tracker.protocols.trackerServer.once(TrackerServer.events.NODE_STATUS_RECEIVED, ({ statusMessage, nodeType }) => {
+        tracker.protocols.trackerServer.once(TrackerServer.events.NODE_STATUS_RECEIVED, ({ statusMessage, _ }) => {
```
Comment (harbu) [inline test/integration/tracker-node-status.test.js]: You can leave out underscore.

File: test/integration/tracker-node-status.test.js [hunk 3]
```diff
@@ -52,7 +52,7 @@ describe('check status message flow between tracker and two nodes', () => {
             })
 
             let receivedTotal = 0
-            tracker.protocols.trackerServer.on(TrackerServer.events.NODE_STATUS_RECEIVED, ({ statusMessage, nodeType }) => {
+            tracker.protocols.trackerServer.on(TrackerServer.events.NODE_STATUS_RECEIVED, ({ statusMessage, _ }) => {
```
Comment (harbu) [inline test/integration/tracker-node-status.test.js]: You can leave out underscore.

File: test/integration/tracker-node-reconnect-instructions.test.js [hunk 4]
```diff
@@ -57,7 +57,7 @@ describe('Check tracker instructions to node', () => {
         })
 
         let receivedTotal = 0
-        tracker.protocols.trackerServer.on(TrackerServer.events.NODE_STATUS_RECEIVED, ({ statusMessage, nodeType }) => {
+        tracker.protocols.trackerServer.on(TrackerServer.events.NODE_STATUS_RECEIVED, ({ statusMessage, _ }) => {
```
Comment (harbu) [inline test/integration/tracker-node-reconnect-instructions.test.js]: You can leave out underscore.

File: test/integration/tracker-node-reconnect-instructions.test.js [hunk 5]
```diff
@@ -48,7 +48,7 @@ describe('Check tracker instructions to node', () => {
         let firstCheck = false
         let secondCheck = false
 
-        otherNodes[1].protocols.nodeToNode.endpoint.once(endpointEvents.PEER_DISCONNECTED, ({ _, reason }) => {
+        otherNodes[1].protocols.nodeToNode.endpoint.once(events.PEER_DISCONNECTED, ({ _, reason }) => {
```
Comment (harbu) [inline test/integration/tracker-node-reconnect-instructions.test.js]: You can leave out underscore.

**Review by harbu — CHANGES_REQUESTED** [review]

**Comment by mirpo** [comment]
moved to #299

**Pull request closed without merging**